### ISSUE TITLE
Add device_id to EventHubEvent

### ISSUE THREAD

**Issue by fbaeuerlein** [Azure/azure-functions-python-worker#288]
Please add a property device_id to EventHubData similar to EventHub reader implementation (https://github.com/Azure/azure-event-hubs-python/blob/master/azure/eventhub/common.py).

**Comment by elprans** [comment]
How does "device_id" get encoded in the host message?  As far as I can see, there is no mention of it in the `EventData` documentation.  What is the best way to test this?   /cc @asavaritayal

**Comment by asavaritayal** [comment]
It should be available as part of the (trigger metadata)[https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-event-hubs#trigger---event-metadata]

Just chatted with @pragnagopa and looks like there's a known issue on the extension itself - https://github.com/Azure/azure-webjobs-sdk/issues/2068

**Comment by fabiocav** [comment]
Duplicate of https://github.com/Azure/azure-webjobs-sdk/issues/2068

**Comment by fabiocav** [comment]
Reopening as there will be Python specific tasks once the linked issue is fixed.

**Comment by asavaritayal** [comment]
@elprans quick update - the issue is now unblocked for you to make the language worker changes.

**Comment by pragnagopa** [comment]
Issue https://github.com/Azure/azure-webjobs-sdk/issues/2068 is fixed. Next runtime release will include the fix.

**Comment by asavaritayal** [comment]
@elprans do you have an update on this?

**Comment by pragnagopa** [comment]
Unfortunately, runtime release got delayed. ETA for complete roll out is end of this week.
